activesupport/lib/active_support/backtrace_cleaner.rb

```diff
@@ -3,33 +3,21 @@
 module ActiveSupport
   # = Backtrace Cleaner
   #
-  # Backtraces often include many lines that are not relevant for the context
-  # under review. This makes it hard to find the signal amongst the backtrace
-  # noise, and adds debugging time. With a BacktraceCleaner, filters and
-  # silencers are used to remove the noisy lines, so that only the most relevant
-  # lines remain.
+  # BacktraceCleaner holds filters and silencers which will inspect and mutate line-by-line the application's backtrace.
```

Comment:
What's the reasoning for replacing the opening paragraph with this single line? I feel like the original does a better job at explaining why the class is useful.